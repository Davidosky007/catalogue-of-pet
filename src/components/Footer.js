import React from 'react';
import { Link } from '@reach/router';

const Footer = () => (
  <footer className="footer__wrapper">
    <Link to="/">PetFinder</Link>
  </footer>
);
export default Footer;
