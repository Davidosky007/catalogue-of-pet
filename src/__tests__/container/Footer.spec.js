import React from 'react';
import '@testing-library/jest-dom';
import renderer from 'react-test-renderer';
import Footer from '../../components/Footer';

test('should render Details components properly ', () => {
  const details = renderer.create(<Footer />);
  const final = details.toJSON();
  expect(final).toMatchSnapshot();
});
