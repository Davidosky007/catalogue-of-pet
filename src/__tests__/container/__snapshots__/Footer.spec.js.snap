// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should render Details components properly  1`] = `
<footer
  className="footer__wrapper"
>
  <a
    aria-current="page"
    href="/"
    onClick={[Function]}
  >
    PetFinder
  </a>
</footer>
`;
